src/QMCDrivers/WFOpt/QMCCostFunction.cpp

```diff
@@ -710,8 +721,8 @@ QMCCostFunction::Return_rt QMCCostFunction::fillOverlapHamiltonianMatrices(Matri
     }
   }
 
-  myComm->allreduce(D_avg);
-
+  myComm->allreduce(D_avg);  
+  
```

Comment:
There some extra spaces introduced in the PR.  Running clang-format would get rid of them.